docs/minio-operator/minio-operator.yaml

```diff
@@ -87,6 +87,6 @@ spec:
       serviceAccountName: minio-operator-sa
       containers:
       - name: minio-operator
-        image: minio/k8s-operator:latest
+        image: mattermost/minio-operator
```

Comment:
Pointing at our Docker image to support standalone Minio until https://github.com/minio/minio-operator/pull/23 is accepted and also until the Minio operator is tagged so we don't have to worry about breaking changes